README.md

```diff
@@ -1,44 +1,44 @@
-# Calcul CFOF via iSAX
+# CFOF score computation via iSAX
 
 [![Documentation Status](https://readthedocs.org/projects/pycfofisax/badge/?version=main)](https://pycfofisax.readthedocs.io/fr/main/?badge=main)
 
 [![PyPI](https://github.com/luk-f/pyCFOFiSAX/actions/workflows/python-publish.yml/badge.svg)](https://github.com/luk-f/pyCFOFiSAX/actions/workflows/python-publish.yml)
 
-Projet thèse Lucas \
+Lucas' PhD projects \
 2017-2020
 
-## Pour commencer
+## To start
 
 ### Installation via PyPI
 
-Lancer `pip install pyCFOFiSAX` ou `python3 -m pip install pyCFOFiSAX`.
+Run `pip install pyCFOFiSAX` or `python3 -m pip install pyCFOFiSAX`.
 
-### Installation manuelle
+### Manual installation
 
-Lancer `pip install -r requirements.txt` ou `python3 -m pip install -r requirements.txt`.
+Run `pip install -r requirements.txt` or `python3 -m pip install -r requirements.txt`.
 
-Le package `tslearn` requière `Cython`, `numba` et `llvmlite` (`pip` téléchargera automatiquement ces packages).
-Cependant pour `Cython`, il est [nécessaire d'avoir un compileur C (comme décrit sur ce site)](https://cython.readthedocs.io/en/latest/src/quickstart/install.html),
-et le [package `llvmlite`](https://llvmlite.readthedocs.io/en/latest/) n'a été, pour ce projet, testé qu'avec Python 3.7 et 3.8
-que nous recommandons.
+The `tslearn` package requires `Cython`, `numba` and `llvmlite` (`pip` will automatically download these packages).
+However for `Cython`, it is [necessary to have a C compiler (as described on this website)](https://cython.readthedocs.io/en/latest/src/quickstart/install.html),
+and the [package `llvmlite`](https://llvmlite.readthedocs.io/en/latest/) for this project, tested only with Python 3.7 and 3.8
+that we recommend.
 
-Sinon, l'utilisation de `conda` simplifie l'installation (comme recommandé sur le site de [tslearn](https://tslearn.readthedocs.io/en/latest/index.html)), car ne nécessite pas 
-l'installation de compileur C et des packages `Cython` et `llvmlite`.
+Otherwise, the use of `conda` simplifies the installation (as recommended on the site of[tslearn](https://tslearn.readthedocs.io/en/latest/index.html)), because does not require
```

Comment:
```suggestion
Otherwise, the use of `conda` simplifies the installation (as recommended on the site of [tslearn](https://tslearn.readthedocs.io/en/stable/)), because does not require
```

Reply:
```suggestion
Otherwise, the use of `conda` simplifies the installation (as recommended on the site of[tslearn](https://tslearn.readthedocs.io/en/latest/index.html)), because it does not require
```